src/Step/Deploy/SetCryptKey.php

```diff
@@ -73,6 +73,11 @@ public function execute()
         $key = $this->environment->getCryptKey();
 
         if (empty($key)) {
+            $this->logger->error(
+                'Crypt key missing. Add the crypt key to the "app/etc/env.php" file, or set the "CRYPT_KEY" '
```

Comment:
@shiftedreality please check that setting this variable via UI works. AFAIK it's not working, since this variable has no reader from UI.

Reply:
It should work, will ask QA to test

Reply:
@andriyShevtsov / @mveeramneni FYI. Please make sure to test this case

Reply:
We recently added a note to the Cloud Guide saying that you had to add the variable through Project Web UI so it wouldn't be exposed in the source repo--is this incorrect? 



![image](https://user-images.githubusercontent.com/6391769/69171211-21c8e000-0ac1-11ea-9caf-08b9dc3e1fe5.png)

https://devdocs.magento.com/guides/v2.2/cloud/env/variables-deploy.html#crypt_key